README.md

```diff
@@ -1,43 +1,61 @@
 # GTC to VCF converter
 
 ## Usage
-```bash
-usage: gtc_to_vcf.py [-h] [--gtc-file GTC_FILE] --manifest-file MANIFEST_FILE
-                     --genome-fasta-file GENOME_FASTA_FILE
-                     [--output-vcf-file OUTPUT_VCF_FILE] [--skip-indels]
-                     [--log-file LOG_FILE] [--expand-identifiers]
-                     [--unsquash-duplicates] [--auxiliary-loci AUXILIARY_LOCI]
+```none
+usage: gtc_to_vcf.py [-h] [--gtc-files GTC_FILES [GTC_FILES ...]]
```

Comment:
I had implemented as a directory at first, but though this might be more generic. If a user does want to specify a directory, they can always do <dir>/*.gtc

Reply:
Good point. It may be fairly obvious, but I could see putting something like "To convert all the GTCs in a directory, run with DIR/*.gtc" in the README helping with use-ability a bit.